### PR TITLE
CHANGELOG updates for Enterprise 2.0

### PR DESCRIPTION
**Pull request by portertech**

### CODE REVIEW

File: docs/0.26/overview/changelog.md [hunk 1]
```diff
@@ -10,11 +10,59 @@ weight: 5
...
 - [Core 0.26.1 Release Notes](#core-v0-26-1)
 - [Core 0.26.0 Release Notes](#core-v0-26-0)
 
+## Enterprise 2.0.0 Release Notes {#enterprise-v2-0-0}
+
+**October 16, 2016** &mdash; Sensu Enterprise version 2.0.0 has been
+	released and is available for immediate download. Please note the
+	following improvements:
+
+### IMPORTANT {#enterprise-v2-0-0-important}
```
Comment (portertech) [inline docs/0.26/overview/changelog.md:27]: Not sure if this is sufficient. Should we duplicate much of the 0.26 "IMPORTANT" CHANGELOG entry?

Comment (cwjohnston) [reply docs/0.26/overview/changelog.md:27]: I think linking to the 0.26 section on this page is sufficient, but I think we're missing a `#` in the inline link below.

File: docs/0.26/overview/changelog.md [hunk 2]
```diff
@@ -10,11 +10,59 @@ weight: 5
...
+## Core 0.26.5 Release Notes {#core-v0-26-5}
+
+Source: [GitHub.com][1]
+
+**October 12, 2016** &mdash; Sensu Core version 0.26.5 has been released
+	and is available for immediate download. Please note the following
+	improvements:
+
+### CHANGES {#core-v0-26-5-changes}
+
```
Comment (cwjohnston) [inline docs/0.26/overview/changelog.md:47]: Missing `Source` link to GitHub?

Comment (portertech) [reply docs/0.26/overview/changelog.md:47]: Ah yes, will update.

File: docs/0.26/overview/changelog.md (hunk 1 above)
Comment (portertech) [reply docs/0.26/overview/changelog.md:27]: Fixed

File: docs/0.26/overview/changelog.md (hunk 2 above)
Comment (portertech) [reply docs/0.26/overview/changelog.md:47]: Updated.

**Comment by cwjohnston** [comment]
🌮

**Pull request merged**